### PR TITLE
Sparsification update

### PR DESCRIPTION
**Pull request by markurtz**
- update sparsification descriptions and move to preferred verbage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,25 +44,32 @@ limitations under the License.
...
-[Keras](https://keras.io/), and [TensorFlow V1](http://tensorflow.org/) ecosystems currently.
+SparseML is integrated for easy model optimizations within the [PyTorch](https://pytorch.org/), [Keras](https://keras.io/), and [TensorFlow V1](http://tensorflow.org/) ecosystems currently.
 
-### Related Products
+## Sparsification
 
-- [DeepSparse](https://github.com/neuralmagic/deepsparse): CPU inference engine that delivers unprecedented performance for sparse models
-- [SparseZoo](https://github.com/neuralmagic/sparsezoo): Neural network model repository for highly sparse models and optimization recipes
-- [Sparsify](https://github.com/neuralmagic/sparsify): Easy-to-use autoML interface to optimize deep neural networks for better inference performance and a smaller footprint
+Sparsification is the process of taking a trained deep learning model and removing redundant information from the over precise and over parameterized network resulting in a faster and smaller model.
```
Comment (jeanniefinks) [inline README.md:54]:
```suggestion
Sparsification is the process of taking a trained deep learning model and removing redundant information from the overprecise and over-parameterized network resulting in a faster and smaller model.
```

File: README.md [hunk 2]
```diff
@@ -44,25 +44,32 @@ limitations under the License.
...
+Techniques for sparsification are all encompassing including everything from inducing sparsity using [pruning](https://neuralmagic.com/blog/pruning-overview/) and [quantization](https://arxiv.org/abs/1609.07061) to enabling naturally occurring sparsity using [activation sparsity](http://proceedings.mlr.press/v119/kurtz20a.html) or [winograd/FFT](https://arxiv.org/abs/1509.09308). 
+When implemented correctly, these techniques result in significantly more performant and smaller models with limited to no effect on the baseline metrics.
+For example, pruning plus quantization can give over [7x improvements in performance](resnet50link) while recovering to nearly the same baseline accuracy.
+
+The DeepSparse product suite builds on top of sparsification enabling you to easily apply the techniques to your datasets/models using recipe driven approaches.
```
Comment (jeanniefinks) [inline README.md:59]:
```suggestion
The DeepSparse product suite builds on top of sparsification enabling you to easily apply the techniques to your datasets and models using recipe-driven approaches.
```

File: README.md [hunk 3]
```diff
@@ -44,25 +44,32 @@ limitations under the License.
...
+Recipes encode the directions for how to sparsify a model into a simple, easily editable format.
+Download a sparsification recipe/sparsified model from the [SparseZoo](https://github.com/neuralmagic/sparsezoo) or create one using [Sparsify](https://github.com/neuralmagic/sparsify), apply it using [SparseML](https://github.com/neuralmagic/sparseml) with only a few lines of code, and deploy with the [DeepSparse Engine](https://github.com/neuralmagic/deepsparse) for unprecedented performance on CPUs.
```
Comment (jeanniefinks) [inline README.md:61]:
```suggestion
- Download a sparsification recipe and sparsified model from the [SparseZoo](https://github.com/neuralmagic/sparsezoo).
- Alternatively, create a recipe for your model using [Sparsify](https://github.com/neuralmagic/sparsify).
- Apply your recipe with only a few lines of code using [SparseML](https://github.com/neuralmagic/sparseml).
- Finally, for GPU-level performance on CPUs, deploy your sparse-quantized model with the [DeepSparse Engine](https://github.com/neuralmagic/deepsparse).


```

File: README.md [hunk 4]
```diff
@@ -44,25 +44,32 @@ limitations under the License.
...
+Visualization of the full product flow:
```
Comment (jeanniefinks) [inline README.md:62]:
```suggestion
**Full DeepSparse product flow:**  
```

File: docs/source/index.rst [hunk 5]
```diff
@@ -58,16 +57,20 @@ and smaller models for the ML performance community at large.
...
+==============
 
-- `DeepSparse <https://github.com/neuralmagic/deepsparse />`_:
-  CPU inference engine that delivers unprecedented performance for sparse models
-- `SparseZoo <https://github.com/neuralmagic/sparsezoo />`_:
-  Neural network model repository for highly sparse models and optimization recipes
-- `Sparsify <https://github.com/neuralmagic/sparsify />`_:
-  Easy-to-use autoML interface to optimize deep neural networks for
-  better inference performance and a smaller footprint
+Sparsification is the process of taking a trained deep learning model and removing redundant information from the over precise and over parameterized network resulting in a faster and smaller model.
```
Comment (jeanniefinks) [inline docs/source/index.rst:63]:
```suggestion
Sparsification is the process of taking a trained deep learning model and removing redundant information from the overprecise and over-parameterized network resulting in a faster and smaller model.
```

File: docs/source/index.rst [hunk 6]
```diff
@@ -58,16 +57,20 @@ and smaller models for the ML performance community at large.
...
+When implemented correctly, these techniques result in significantly more performant and smaller models with limited to no effect on the baseline metrics.
+For example, pruning plus quantization can give over [7x improvements in performance](resnet50link) while recovering to nearly the same baseline accuracy.
+
+The DeepSparse product suite builds on top of sparsification enabling you to easily apply the techniques to your datasets/models using recipe driven approaches.
+Recipes encode the directions for how to sparsify a model into a simple, easily editable format.
+Download a sparsification recipe/sparsified model from the `SparseZoo <https://github.com/neuralmagic/sparsezoo>`_ or create one using `Sparsify <https://github.com/neuralmagic/sparsify>`_, apply it using `SparseML <https://github.com/neuralmagic/sparseml>`_ with only a few lines of code, and deploy with the `DeepSparse Engine <https://github.com/neuralmagic/deepsparse>`_ for unprecedented performance on CPUs.
+Visualization of the full product flow:
```
Comment (jeanniefinks) [inline docs/source/index.rst:71]:
```suggestion
**Full DeepSparse product flow:**  
```

File: docs/source/index.rst [hunk 7]
```diff
@@ -58,16 +57,20 @@ and smaller models for the ML performance community at large.
...
-  Easy-to-use autoML interface to optimize deep neural networks for
-  better inference performance and a smaller footprint
+Sparsification is the process of taking a trained deep learning model and removing redundant information from the over precise and over parameterized network resulting in a faster and smaller model.
+Techniques for sparsification are all encompassing including everything from inducing sparsity using `pruning <https://neuralmagic.com/blog/pruning-overview/>`_ and `quantization <https://arxiv.org/abs/1609.07061>`_ to enabling naturally occurring sparsity using `activation sparsity <http://proceedings.mlr.press/v119/kurtz20a.html>`_ or `winograd/FFT <https://arxiv.org/abs/1509.09308>`_.
+When implemented correctly, these techniques result in significantly more performant and smaller models with limited to no effect on the baseline metrics.
+For example, pruning plus quantization can give over [7x improvements in performance](resnet50link) while recovering to nearly the same baseline accuracy.
+
+The DeepSparse product suite builds on top of sparsification enabling you to easily apply the techniques to your datasets/models using recipe driven approaches.
```
Comment (jeanniefinks) [inline docs/source/index.rst:68]:
```suggestion
The DeepSparse product suite builds on top of sparsification enabling you to easily apply the techniques to your datasets and models using recipe-driven approaches.
```

File: docs/source/index.rst [hunk 8]
```diff
@@ -58,16 +57,20 @@ and smaller models for the ML performance community at large.
...
+Recipes encode the directions for how to sparsify a model into a simple, easily editable format.
+Download a sparsification recipe/sparsified model from the `SparseZoo <https://github.com/neuralmagic/sparsezoo>`_ or create one using `Sparsify <https://github.com/neuralmagic/sparsify>`_, apply it using `SparseML <https://github.com/neuralmagic/sparseml>`_ with only a few lines of code, and deploy with the `DeepSparse Engine <https://github.com/neuralmagic/deepsparse>`_ for unprecedented performance on CPUs.
```
Comment (jeanniefinks) [inline docs/source/index.rst:70]:
```suggestion
- Download a sparsification recipe and sparsified model from the [SparseZoo](https://github.com/neuralmagic/sparsezoo).
- Alternatively, create a recipe for your model using [Sparsify](https://github.com/neuralmagic/sparsify).
- Apply your recipe with only a few lines of code using [SparseML](https://github.com/neuralmagic/sparseml).
- Finally, for GPU-level performance on CPUs, deploy your sparse-quantized model with the [DeepSparse Engine](https://github.com/neuralmagic/deepsparse).


```

File: docs/source/recipes.md [hunk 9]
```diff
@@ -27,9 +27,7 @@ and apply the modifications to the model and training pipeline.
 
 In a recipe, modifiers must be written in a list that includes "modifiers" in its name.
 
-The easiest ways to get or create optimization recipes are by either using 
-the pre-configured recipes in [SparseZoo](https://github.com/neuralmagic/sparsezoo) or 
-using [Sparsify's](https://github.com/neuralmagic/sparsify) autoML style creation.
+The easiest ways to get or create recipes are by either using the pre-configured recipes in [SparseZoo](https://github.com/neuralmagic/sparsezoo) or using [Sparsify's](https://github.com/neuralmagic/sparsify) automatic creation.
 
 However, power users may be inclined to create their own recipes by hand to enable more 
 fine grained control or to add in custom modifiers.
```
Comment (jeanniefinks) [inline docs/source/recipes.md:33]:
```suggestion
fine-grained control or to add in custom modifiers.
```

**Review by NMsasa — COMMENTED** [review]
Missing the last sentence about "this repo.." : This repository contains integrations within the PyTorch, Keras, and TensorFlow V1 ecosystems, allowing for seamless model sparsification.

**Review by NMsasa — APPROVED** [review]

**Review by jeanniefinks — APPROVED** [review]

**Review by bfineran — APPROVED** [review]

**Pull request merged**